packages/abstraxion/src/hooks/useAbstraxionSigningClient.ts

```diff
@@ -22,6 +22,10 @@ export const useAbstraxionSigningClient = () => {
         if (!abstraxionAccount) {
           throw new Error("No account found.");
         }
+
+        if (!grantorAddress) {
+          throw new Error("No grantor found.");
```

Comment:
TIL
https://english.stackexchange.com/questions/34908/when-is-the-correct-time-to-use-granter-or-grantor